api/managementpb/actions.proto

```diff
@@ -0,0 +1,152 @@
+syntax = "proto3";
+
+package management;
+
+// This option is used to set the default error response for all Swagger APIs generated from this file.
+// See also header.json.
+// It has to be copy&pasted into each file.
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+  responses: {
+    key: "default"
+    value: {
+      description: "An error response."
+      schema: {
+        json_schema: {
+          ref: "#/definitions/managementpbErrorResponse"
+        }
+      }
+    }
+  }
+};
+option go_package = "managementpb";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+// ActionType represents Action type.
+enum ActionType {
+  ACTION_TYPE_INVALID = 0;
+  PT_SUMMARY = 1;
+  PT_MYSQL_SUMMARY = 2;
+  MYSQL_EXPLAIN = 8;
+}
+
+message GetActionRequest {
+  // Unique Action ID.
+  string action_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+}
+
+message GetActionResponse {
+  // Unique Action ID.
+  string action_id = 1;
+  // pmm-agent ID where this Action is running / was run.
+  string pmm_agent_id = 2;
+  // Current Action output; may be partial if Action is still running.
+  string output = 3;
+  // True if Action is finished.
+  bool done = 4;
+  // Error message if Action failed.
+  string error = 5;
+}
+
+message StartPTSummaryActionRequest {
+  // pmm-agent ID where to run this Action.
+  string pmm_agent_id = 1;
+  // Node ID for this Action.
+  string node_id = 2;
+  //
+  // TODO pt-summary-specific parameters if we need them.
+
+}
+
+message StartPTSummaryActionResponse {
+  // Unique Action ID.
+  string action_id = 1;
+  // pmm-agent ID where to this Action was started.
+  string pmm_agent_id = 2;
+}
+
+message StartPTMySQLSummaryActionRequest {
+  // pmm-agent ID where to run this Action.
+  string pmm_agent_id = 1;
+  // Service ID for this Action.
+  string service_id = 2;
```

Comment:
this should be required param, isn't it?

Reply:
oh it should not be part of this PR

Reply:
did you implement this action?

Reply:
No, I don't there is a stub. I just merged @AlekSi API as he wants in my branch, so this is not my API at all.

Reply:
>this should be required param, isn't it?

Yes, it should be. Good catch.